### PR TITLE
removed cyclic interactions with Enchanting Awakened

### PR DESCRIPTION
**Pull request by jmarsh411**

### CODE REVIEW

**Comment by Ortham** [comment]
So is the problem just that the `req` fields need to load before the plugin they're attached to, but the patches needs to load after?

**Comment by jmarsh411** [comment]
Yes, but they're mostly 3-plugin cycles. For example, previously in the masterlist Apocalypse `required` the `EnchantingAwakenedApocalypsePatch.esp` patch if `EnchantingAwakened.esp` was present. However this would cause Loot to think the patch needs to be loaded first. `EnchantingAwakened.esp` has a `load after` for Apocalypse. I think this `load after` is appropriate here. But all together Loot sees that  `Apocalypse` needs to be loaded after `EnchantingAwakenedApocalypsePatch` from the `req` but EnchantingAwakenedApocalypsePatch needs to be loaded after its master, `EnchantingAwakened`, which needs to be loaded after `Apocalypse`.

I couldn't resolve these with userlist edits because I couldn't delete the `req` in Loot's interface. But these are hardly requirements to begin with in my opinion. Without the patches there are only minor things that won't work as both mods advertise. For example the Soul Trap Cloak that Apocalypse adds won't give Enchanting experience as Enchanting Awakened advertises without the patch. I think requires should be reserved for things that the mod absolutely won't work without, like a MCM mod requiring SkyUI.

**Comment by Ortham** [comment]
OK, sounds good, my only other concern would be that the repeated messages might be better replaced by a generic aliased message in the `common` section, or maybe a new message created there and used, but I'll check that this evening.

File: masterlist.yaml [hunk 1]
```diff
@@ -2469,10 +2469,12 @@ plugins:
         <<: *dirtyPlugin
         itm: 3
   - name: 'Chesko_Frostfall.esp'
-    req:
-      - name: 'EnchantingAwakenedFrostfallPatch.esp'
-        display: '[Enchanting Awakened Frostfall Patch](http://www.nexusmods.com/skyrim/mods/42796)'
-        condition: 'file("EnchantingAwakened.esp")'
+    msg:
```
Comment (Ortham) [inline masterlist.yaml:2477]: Please make this a common message in the `common` root list, e.g.

```
- &alsoUseX
  type: say
  content:
    - lang: en
      text: 'It is highly recommended that you also use %1%.'
```

then use that message like:

```
msg:
  - <<: *alsoUseX
    condition: 'file("EnchantingAwakened.esp") and not file("EnchantingAwakenedApocalypsePatch.esp")'
    subs: [ 'the [Enchanting Awakened Apocalypse Patch](http://www.nexusmods.com/skyrim/mods/42796)' ]
```

and similar for the messages you've added below.

**Review by Ortham — CHANGES_REQUESTED** [review]

**Comment by jmarsh411** [comment]
Makes sense. I've made the changes.

I can also update the common message with the translations from the Unofficial patches since that's where I copied the message from. It would make sense to make them use the new common message too. Let me know whether this should be done in a separate pull request or if you want someone more experienced to do it because of how impactful the change would be.

**Review by Ortham — APPROVED** [review]

**Comment by Ortham** [comment]
:+1:

**Pull request merged**